glslc/src/file_compiler.h

```diff
@@ -16,20 +16,23 @@
 #define GLSLC_FILE_COMPILER_H
 
 #include <string>
-#include "libshaderc_util/compiler.h"
+
+#include "libshaderc_util/string_piece.h"
+#include "libshaderc_util/file_finder.h"
+#include "shaderc.hpp"
```

Comment:
The 'target_link_libraries(glslc PRIVATE ... shaderc)' does the work. This brings shaderc's public include directories to glslc.